Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -217,25 +320,26 @@ void MemoryViewWidget::Update()
 
     // Update column width
     for (int i = starting_column; i < starting_column + column_count - 1; i++)
-      setColumnWidth(i, m_font_width * GetCharacterCount(left_type));
+      m_table->setColumnWidth(i, m_font_width * GetCharacterCount(left_type));
 
     // Extra spacing between dual views.
-    setColumnWidth(starting_column + column_count - 1,
-                   m_font_width * (GetCharacterCount(left_type) + 2));
+    m_table->setColumnWidth(starting_column + column_count - 1,
+                            m_font_width * (GetCharacterCount(left_type) + 2));
 
     starting_column += column_count;
   }
 
   UpdateColumns(m_type, starting_column);
   UpdateBreakpointTags();
 
-  setColumnWidth(0, rowHeight(0));
+  m_table->setColumnWidth(0, m_table->rowHeight(0));
 
-  for (int i = starting_column; i <= columnCount(); i++)
-    setColumnWidth(i, m_font_width * GetCharacterCount(m_type));
+  for (int i = starting_column; i <= m_table->columnCount(); i++)
+    m_table->setColumnWidth(i, m_font_width * GetCharacterCount(m_type));
 
-  viewport()->update();
-  update();
+  m_table->viewport()->update();
+  m_table->update();
+  this->update();
```

Comment:
Just curious, was the `this` added for a particular reason?

Reply:
No particular reason, no. I've removed it again.